lib/FusionInventory/Agent/Task/Wmi.pm

```diff
@@ -0,0 +1,91 @@
+package FusionInventory::Agent::Task::Wmi;
```

Comment:
as WMI is an acronym, I'd rather keep it uppercased: package FusionInventory::Agent::Task::WMI

Reply:
Okay, I agree.